vignettes/mmrm_review_methods.Rmd

```diff
@@ -0,0 +1,277 @@
+
+### SAS
+<pre><code>PROC MIXED DATA = fev_data cl method=reml;
+CLASS RACE(ref = 'Asian') AVISIT(ref = 'VIS4') SEX(ref = 'Male') ARMCD(ref = 'PBO') USUBJID;
+MODEL FEV1 = RACE SEX ARMCD AVISIT ARMCD*AVISIT / ddfm=satterthwaite solution chisq;
+<b>REPEATED AVISIT / subject=USUBJID type=un</b> r rcorr;
+<b>WEIGHT WEIGHT</b>;
+</code></pre>
+  
+### mmrm
+<pre><code>mmrm::mmrm(formula = FEV1 ~ RACE + SEX + ARMCD * AVISIT + <b>us(AVISIT | USUBJID)</b>,
+  data = fev_data, <b>weights = WEIGHT</b>)
+</code></pre>
+
+## Toeplitz (heterogeneous)
+
+### SAS
+<pre><code>PROC MIXED DATA = fev_data cl method=ml;
+CLASS AVISIT(ref = 'VIS4') USUBJID;
+MODEL FEV1 = / ddfm=satterthwaite solution chisq;
+<b>REPEATED AVISIT / subject=USUBJID type=TOEPH</b> r=1, 2, 3, 4, 5, 6, 7, 8, 9, 10 rcorr;
+</code></pre>
+
+### mmrm
+<pre><code>mmrm::mmrm(formula = FEV1 ~ <b>toeph(AVISIT | USUBJID)</b>,
+  data = fev_data, reml = FALSE)
+</code></pre>
+
+## Toeplitz (grouped heterogeneous)
+### SAS
+<pre><code>PROC MIXED DATA = fev_data cl method=ml;
+CLASS AVISIT(ref = 'VIS4') USUBJID ARMCD;
+MODEL FEV1 = / ddfm=satterthwaite solution chisq;
+<b>REPEATED AVISIT / subject=USUBJID type=TOEPH</b> r=1, 2, 3, 4, 5, 6, 7, 8, 9, 10 rcorr <b>group=ARMCD</b>;
+</code></pre>
+
+### mmrm
+<pre><code>mmrm::mmrm(formula = FEV1 ~ <b>toeph(AVISIT | ARMCD / USUBJID)</b>,
+  data = fev_data, reml = FALSE)
+</code></pre>
+
+## Toeplitz (homogeneous)
+### SAS
+<pre><code>PROC MIXED DATA = fev_data cl method=ml;
+CLASS AVISIT(ref = 'VIS4') USUBJID;
+MODEL FEV1 = / ddfm=satterthwaite solution chisq;
+<b>REPEATED AVISIT / subject=USUBJID</b> <b>type=TOEP</b> r=1, 2, 3, 4, 5, 6, 7, 8, 9, 10 rcorr;
+</code></pre>
+
+### mmrm
+<pre><code>mmrm::mmrm(formula = FEV1 ~ <b>toep(AVISIT | USUBJID)</b>, 
+  data = fev_data, reml = FALSE)
+</code></pre>
+
+## Spatial exponential
+
+### SAS 
+<pre><code>PROC MIXED DATA = fev_data cl method=ml;
+CLASS USUBJID;
+MODEL FEV1 = / ddfm=satterthwaite solution chisq;
+<b>REPEATED / subject=USUBJID type=sp(exp)(visitn)</b> rcorr;
+</code></pre>
+
+### mmrm
+<pre><code>mmrm::mmrm(formula = FEV1 ~ <b>sp_exp(VISITN | USUBJID)</b>, 
+  data = fev_data, reml = FALSE)
+</code></pre>
+
+## Compound symmetry (heterogeneous)
+
+### SAS
+<pre><code>PROC MIXED DATA = fev_data cl method=ml;
+CLASS AVISIT(ref = 'VIS4') USUBJID;
+MODEL FEV1 = / ddfm=satterthwaite solution chisq;
+<b>REPEATED AVISIT / subject=USUBJID type=CSH</b> r=1, 2, 3, 4, 5, 6, 7, 8, 9, 10 rcorr;
+</code></pre>
+
+### mmrm
+<pre><code>mmrm::mmrm(formula = FEV1 ~ <b>csh(VISITN | USUBJID)</b>, 
+  data = fev_data, reml = FALSE)
+</code></pre>
+
+### nlme::gls
+<pre><code>nlme::gls(formula = FEV1 ~  SEX + ARMCD * AVISIT,
+  data = fev_data,
+  correlation = <b>nlme::corCompSymm(form = ~AVISIT | USUBJID)</b>,
+  weights = nlme::varIdent(form = ~1|AVISIT),
+  na.action = na.omit)
+</code></pre>
+
+## Compound symmetry (homogeneous)
+
+### SAS
+<pre><code>PROC MIXED DATA = fev_data cl method=ml;
```

Comment:
ah I guess we need to have these html tags to be able to bold face stuff inside?

Reply:
yes, that's correct. markdown code highlighting and html bolding are mutually exclusive as far as I have found.